Sources/Automerge/Document.swift

```diff
@@ -768,6 +768,61 @@ public final class Document: @unchecked Sendable {
         }
     }
 
+    /// Get the list of marks for a text object at ``Cursor``.
+    ///
+    /// Use:
```

Comment:
Since this is so nearly identical to `MarksAt(obj:position:heads)`, it would be good to include some detail on why a developer should use this method compared to a straight index position. The example above shows the how, but is identifical to the position example in that nothing is changing in the Text object that the cursor would properly track in terms of location over time.

Reply:
Oh good call - I will do

Reply:
https://github.com/automerge/automerge-swift/pull/186#discussion_r1637157757